bin/kontrast

```diff
@@ -77,7 +78,7 @@ module Kontrast
         end
 
         private
-            def load_config(config)
+            def load_config(config, specs_path = nil)
```

Comment:
Per our code conventions this should be a named arg, and it probably should be here too.